doc/en/user/source/extensions/geofence-server/installing.rst

```diff
@@ -3,7 +3,7 @@
 Installing the GeoServer GeoFence Server extension
 ==================================================
 
- #. Download the extension from the `nightly GeoServer community module builds <http://ares.opengeo.org/geoserver/master/community-latest/>`_.
+ #. Download the extension from the `nightly GeoServer community module builds <https://build.geoserver.org/geoserver/master/ext-latest/>`_.
```

Comment:
Well since GeoFence is an extension now, the text should be updated too, usually for extensions something like this is used:
```

#. Navigate to the `GeoServer download page <http://geoserver.org/download>`_.
#. Find the page that matches the version of the running GeoServer.
#. Download the GeoFence extension. The download link will be in the :guilabel:`Extensions` 
   section under :guilabel:`Other`.
#. Extract the files in this archive to the :file:`WEB-INF/lib` directory of your GeoServer installation.
#. Restart GeoServer
```

Although since GeoFence is only available in master as an extension I'm not sure where it will appear on the download page,  @afabiani any idea ?

Reply:
@nmco It does not appear on previous versions. We need to point to `master/ext-latest/` right now.